### PR TITLE
Add instructions to aws/awscli install and some further contextual in…

### PR DESCRIPTION
**Pull request by freytheviking**
Added the following to `README.md`

1) A Step 0 to install `aws` and `awscli` python package.
2) Additional prompt to ask Michael G. for AWS access.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,19 @@ Use the bash script provided and provide a version number.
...
+The above code snippet will not work until you have your AWS credentials. Please request access from Michael G.
+
 ## How to use the image
 
+#### Step 0: Install `aws` and `awscli` python package
+
+Install the following packages if you don't already
+
+```
+pip install aws
```
Comment (rjackson90) [inline README.md]:
```suggestion
```
I think we just needed `awscli`

File: README.md [hunk 2]
```diff
@@ -8,8 +8,19 @@ Use the bash script provided and provide a version number.
 source build_and_push_image.sh ${MAJOR}.${MINOR}.${PATCH}
 ```
 
+The above code snippet will not work until you have your AWS credentials. Please request access from Michael G.
```
Comment (rjackson90) [inline README.md]:
```suggestion
The above code snippet will not work until you have your AWS credentials.
```
Making this change allows these docs to remain useful even if the procedure for obtaining AWS creds changes in the future. For example, the process could be delegated to other staff members or we might create an automated process which provisions credentials ahead of time.

**Review by rjackson90 — COMMENTED** [review]
Thanks for updating the docs! Being explicit about dependencies is a pit of success.

File: README.md (hunk 2 above)
Comment (freytheviking) [reply README.md]: Makes sense.

File: README.md (hunk 1 above)
Comment (freytheviking) [reply README.md]: Got ya!

**Pull request merged**